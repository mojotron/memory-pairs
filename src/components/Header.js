import "../styles/Header.css";
const Header = () => {
  return (
    <header className="Header">
      <h1 className="Header__heading">Memory Pairs</h1>
      <p className="Header__info">
        find all pairs and get your name on leader board
      </p>
    </header>
  );
};

export default Header;
