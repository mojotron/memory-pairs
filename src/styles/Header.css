.Header {
  height: 10vh;
  background-color: var(--orange);
}
