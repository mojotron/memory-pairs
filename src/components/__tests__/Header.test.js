import { render, screen } from "@testing-library/react";
import Header from "../Header";

describe("Header component", () => {
  test("Header snapshot", () => {
    const { container } = render(<Header />);
    expect(container).toMatchSnapshot();
  });
});
