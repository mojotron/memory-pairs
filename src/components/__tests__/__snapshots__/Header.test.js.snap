// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component Header snapshot 1`] = `
<div>
  <header
    class="Header"
  >
    <h1
      class="Header__heading"
    >
      Memory Pairs
    </h1>
    <p
      class="Header__info"
    >
      find all pairs and get your name on leader board
    </p>
  </header>
</div>
`;
